src/scripts/videoosd.js

```diff
@@ -1,732 +1,1336 @@
+      var html;
+
+      if (date) {
+        date = datetime.parseISO8601Date(date);
+        html = getDisplayTimeWithoutAmPm(date);
+      }
+
+      elem.innerHTML = html || "";
+    }
+
+    function shouldEnableProgressByTimeOfDay(item) {
+      return !("TvChannel" !== item.Type || !item.CurrentProgram);
+    }
 
-        function stopOsdHideTimer() {
-            osdHideTimeout && (clearTimeout(osdHideTimeout), osdHideTimeout = null)
+    function updateNowPlayingInfo(player, state) {
+      var item = state.NowPlayingItem;
+
+      if (currentItem = item, !item) {
+        setPoster(null);
+        updateRecordingButton(null);
+        Emby.Page.setTitle("");
+        nowPlayingVolumeSlider.disabled = true;
+        nowPlayingPositionSlider.disabled = true;
+        btnFastForward.disabled = true;
+        btnRewind.disabled = true;
+        view.querySelector(".btnSubtitles").classList.add("hide");
+        view.querySelector(".btnAudio").classList.add("hide");
+        view.querySelector(".osdTitle").innerHTML = "";
+        return void (view.querySelector(".osdMediaInfo").innerHTML = "");
+      }
+
+      enableProgressByTimeOfDay = shouldEnableProgressByTimeOfDay(item);
+      getDisplayItem(item).then(updateDisplayItem);
+      nowPlayingVolumeSlider.disabled = false;
+      nowPlayingPositionSlider.disabled = false;
+      btnFastForward.disabled = false;
+      btnRewind.disabled = false;
+
+      if (playbackManager.subtitleTracks(player).length) {
+        view.querySelector(".btnSubtitles").classList.remove("hide");
+      } else {
+        view.querySelector(".btnSubtitles").classList.add("hide");
+      }
+
+      if (playbackManager.audioTracks(player).length > 1) {
+        view.querySelector(".btnAudio").classList.remove("hide");
+      } else {
+        view.querySelector(".btnAudio").classList.add("hide");
+      }
+    }
+
+    function setTitle(item, parentName) {
+      var url = logoImageUrl(item, connectionManager.getApiClient(item.ServerId), {});
+
+      if (url) {
+        Emby.Page.setTitle("");
+        var pageTitle = document.querySelector(".pageTitle");
+        pageTitle.style.backgroundImage = "url('" + url + "')";
+        pageTitle.classList.add("pageTitleWithLogo");
+        pageTitle.classList.remove("pageTitleWithDefaultLogo");
+        pageTitle.innerHTML = "";
+      } else {
+        Emby.Page.setTitle(parentName || "");
+      }
+
+      var documentTitle = parentName || (item ? item.Name : null);
+
+      if (documentTitle) {
+        document.title = documentTitle;
+      }
+    }
+
+    function setPoster(item, secondaryItem) {
+      var osdPoster = view.querySelector(".osdPoster");
+
+      if (item) {
+        var imgUrl = seriesImageUrl(item, {
+          type: "Primary"
+        }) || seriesImageUrl(item, {
+          type: "Thumb"
+        }) || imageUrl(item, {
+          type: "Primary"
+        });
+
+        if (!imgUrl && secondaryItem && (imgUrl = seriesImageUrl(secondaryItem, {
```

Comment:
This change appears to break something.

Testing these by finding the code and manually pasting in your changes since these don't appear to remove all of the lines you mean to.

Chrome says "uncaught promise" and Firefox says "SyntaxError: unexpected token: ')'"

Code that I replaced is:
```js
if (!imgUrl && secondaryItem && (imgUrl = seriesImageUrl(secondaryItem, {
    type: "Primary"
}) || seriesImageUrl(secondaryItem, {
    type: "Thumb"
}) || imageUrl(secondaryItem, {
    type: "Primary"
})), imgUrl) {
    return void (osdPoster.innerHTML = '<img src="' + imgUrl + '" />');
}
```

Exact code I'm using is:
```js
    if (!imgUrl && secondaryItem) {
        imgUrl = seriesImageUrl(secondaryItem, {type: "Primary"})
            || seriesImageUrl(secondaryItem, {type: "Thumb"})
            || imageUrl(secondaryItem, {type: "Primary"});
    if (imgUrl) {
        osdPoster.innerHTML = '<img src="' + imgUrl + '" />';
        return;
    }
}
```

Guessing I've screwed up a bracket somewhere, but I'm not sure where

Reply:
Leave it be then. I cannot find any issue in your replacement code (all brackets seem to be fine).